vnext/Shared/OInstance.cpp

```diff
@@ -607,7 +607,7 @@ std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules
             m_devSettings->debugBundlePath,
             m_devSettings->platformName,
             m_devSettings->bundleAppId,
-            true /*dev*/,
+            m_devSettings->devBundle,
             m_devSettings->useFastRefresh,
```

Comment:
Should we throw some kind of warning when devBundle==false and useFastRefresh==true .. I hope the experience will still run with FastRefresh silently failing .. If there is a fatal error we should probably override FastRefresh to false if devBundle is false ..

Reply:
It works fine as is.